ui/components/app/gas-details-item/gas-details-item.js

```diff
@@ -95,8 +113,14 @@ const GasDetailsItem = ({ userAcknowledgedGasMissing = false }) => {
       }
       subTitle={
         <GasTiming
-          maxPriorityFeePerGas={maxPriorityFeePerGas.toString()}
-          maxFeePerGas={maxFeePerGas.toString()}
+          maxPriorityFeePerGas={(
+            maxPriorityFeePerGas ||
+            hexWEIToDecGWEI(transactionData.txParams.maxPriorityFeePerGas)
+          ).toString()}
+          maxFeePerGas={(
+            maxFeePerGas ||
+            hexWEIToDecGWEI(transactionData.txParams.maxFeePerGas)
+          ).toString()}
```

Comment:
Hey @NiranjanaBinoy : I am not sure why this component needed change. Can you plz detail.

Reply:
For the gas display in send component, `transactionData.txParams` have the data from the draftTransaction.